PSGSuite/Public/Drive/Get-GSDriveFileList.ps1

```diff
@@ -88,8 +88,8 @@ function Get-GSDriveFileList {
         }
         if ($ParentFolderId) {
             if ($Filter) {
-                $Filter += "'$ParentFolderId' in parents"
-                $PSBoundParameters['Filter'] += "'$ParentFolderId' in parents"
+                $Filter = "($Filter) and '$ParentFolderId' in parents"
+                $PSBoundParameters['Filter'] = "($Filter) and '$ParentFolderId' in parents"
```

Comment:
Honestly I never realized you could send an array to the filter parameter!

But, being fixed this way can have, what I believe, is an unintended result.

Consider sending a Filter and Parent, such as this:

`Get-GSDriveFileList -Filter "mimeType contains 'image/' or mimeType contains 'video/'" -ParentFolderId '1WScximqtSunuOWTVXWEnnxyw8Ra0jLty' -Verbose`

which will now result in a final filter like:

![image](https://user-images.githubusercontent.com/48518449/55677073-6a73b600-58af-11e9-9e91-f28358e4f2a1.png)

Because `and` and `or` operators are evaluated from left to right, without parentheses the above example would return only videos containing parent 123456, but would return all images, without regard to their parent.

I believe somehow the filter string(s) need to be surrounded in parentheses so that the parent restriction applies to anything in the filter string.

This is pointed out in discussion of compound clauses in the API example found here:

[https://developers.google.com/drive/api/v3/search-parameters](https://developers.google.com/drive/api/v3/search-parameters)

Reply:
I totally see your point, but ultimately it's up to the user to craft the filter correctly according to Google's syntax. The assumption made here is that, if you're passing multiple filter strings, you're fine with combining them with `and`. If you want something different, craft your filter accordingly, i.e.:

```powershell
PS C:\GDrive\PSModules\PSGSuite> $f = Get-GSDriveFileList -Filter "(name -eq 'file1.txt' -or name -eq 'my other document.txt')" -ParentFolderId '0Bw_1h5ymWhc5V2pWY3E2aVFTZla' -Verbose
VERBOSE: Getting all Drive Files for User 'user@domain.com' matching Filter: (name='file1.txt' or name='my other document.txt') and '0Bw_1h5ymWhc5V2pWY3E2aVFTZla' in parent
```

Reply:
Also worth noting the parameter help contents which link to Google's syntax doc. Also noted is the conversion of PowerShell syntax to Google search syntax being best effort, i.e. `-eq` > `=`:

![image](https://user-images.githubusercontent.com/12724445/55677465-9aba5500-58ad-11e9-8a04-5b22fbdb213c.png)

Reply:
I see your point, and it's easy enough to add the parentheses around any `or` operators within the filter.

Thank you!

Reply:
You bet!